lg_activity/src/lg_activity/activity.py

```diff
@@ -260,6 +268,52 @@ def _is_activity_active(self):
             self.callback(self.topic, state=False, strategy='activity')
             return False
 
+    def _is_message_active(self):
+        """keep last message. Empty list returns False TODO add other cases, empty string, null, ...""" 
+        if self.messages:
+             try:
+                 self.messages = [self.messages[-1]]  # skip to last message and keep it
+                 if self.messages[0]['overlays']:  # TODO add set False methods, else True
+                     self.callback(self.topic, state=True, strategy='message', delay=1)
+                     logger.debug("Setting strems state True")
+                     return True
+             except (IndexError, KeyError, ValueError, TypeError) as e:
+                 logger.debug("error checking stream messages: %" % e)
```

Comment:
`logger.exception` (I think otherwise .error) in an exception block would be good